src/tests/reconnect.test.ts

```diff
@@ -0,0 +1,76 @@
+import FakeTimers from '@sinonjs/fake-timers'
+import test from 'ava'
+import Docker from 'dockerode'
+import { createClient } from 'redis'
+import transporter from '../index.js'
+
+const emit = () => undefined
+
+test.serial(
```

Comment:
This can in fact run in parallel with the other tests now as it has it's "own" redis container in docker compose